cloud/src/meta-service/meta_service.cpp

```diff
@@ -1685,7 +1685,7 @@ static bool check_delete_bitmap_lock(MetaServiceCode& code, std::string& msg, st
     std::string lock_key = meta_delete_bitmap_update_lock_key({instance_id, table_id, -1});
     std::string lock_val;
     DeleteBitmapUpdateLockPB lock_info;
-    auto err = txn->get(lock_key, &lock_val);
+    auto err = txn->get(lock_key, &lock_val, lock_id == -1 /*snapshot*/);
```

Comment:
We should add some unit/p0 tests to verify this branch.

At least, the following test case should be added
![image](https://github.com/user-attachments/assets/efdd633c-f4f3-433a-ab29-9c272f32232c)

Reply:
Besides, the following cases should also be added
* compaction-load conflict
* compaction-compaction conflict
* ??? load-compaction conflict